en/docs/api-analytics/samples/publishing-analytics-events-to-external-systems.md

```diff
@@ -1,250 +1,276 @@
 
-In case of a failure with the above command, use below to set up assets,
+        ```
+        filebeat -e
+        ```
 
-    filebeat -e
+3. [Start](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#start) Filebeat.
 
-[Start](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#start) Filebeat
+### Step 3.2 - View analytics data on Kibana
 
-### View Data on Kibana
+Filebeat comes with pre-built Kibana dashboards and UIs for visualizing log data. 
 
-Filebeat comes with pre-built Kibana dashboards and UIs for visualizing log data.
+#### Step 3.2.1 - Configure the visualization
 
-[Launch Kibana and discover](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#view-data) log data.
+     [Launch Kibana and discover](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#view-data) log data.
 
-Once you have followed and completed the above steps successfully, you will be able to visualize log data as shown below,
+     Once you have followed and completed the above steps successfully, you will be able to visualize log data as follows.
 
-[![Logs listed in kibana]({{base_path}}/assets/img/analytics/samples/logs-listed-in-kibana.png)]({{base_path}}/assets/img/analytics/samples/logs-listed-in-kibana.png)
+     [![Logs listed in kibana]({{base_path}}/assets/img/analytics/samples/logs-listed-in-kibana.png)]({{base_path}}/assets/img/analytics/samples/logs-listed-in-kibana.png)
 
-#### Filtering Total Analytics Traffic
+#### Step 3.2.2 - Filter total analytics traffic
 
-It is possible to view the analytics traffic by applying a filter as shown below,
+The total analytics traffic can be visualized by applying a filter as follows:
 
-Replace `<MetricReporterImplClass>` with the class name given to the MetricReporter implementation class that you have created with your sample.
+Replace `<MetricReporterImplClass>` with the class name given to the `MetricReporter` implementation class that you have created with your sample.
 
 [![Total analytics traffic filter]({{base_path}}/assets/img/analytics/samples/total-analytics-traffic-filter.png)]({{base_path}}/assets/img/analytics/samples/total-analytics-traffic-filter.png)
 
-Once this filter is applied you will be able to visualize analytics traffic as shown below,
+After applying this filter you will be able to visualize analytics traffic as shown below.
 
-[![Filtered total Analytics traffic]({{base_path}}/assets/img/analytics/samples/total-analytics-traffic.png)]({{base_path}}/assets/img/analytics/samples/total-analytics-traffic.png)
+[![Filtered total analytics traffic]({{base_path}}/assets/img/analytics/samples/total-analytics-traffic.png)]({{base_path}}/assets/img/analytics/samples/total-analytics-traffic.png)
 
-#### Filtering Analytics Traffic for a Specific API
+#### Step 3.2.3 - Optionally, filter analytics traffic for a specific API
```

Comment:
@lasanthaS , I made this step optional. Pls verify

Reply:
+1